zerver/webhooks/airbrake/doc.html

```diff
@@ -1,26 +0,0 @@
-<p>Get Zulip notifications for your Airbrake bug tracker!</p>
-
-<p>
-    First, create the stream you'd like to use for Airbrake notifications, and
-    subscribe all interested parties to this stream. We recommend the
```

Comment:
And yes its going to be Bugdown, I think. I could be wrong. @timabbott thoughts?